app/views/exchanges/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<nav class="navbar" style="background-color: #3778c2;" role="navigation">
+  <div class="navbar-brand">
+    <a href="/" class="navbar-item">
+      <span class="icon is-large has-text-white" style="margin-left: 10px;">
+        <i class="fas fa-arrow-left"></i>
+      </span>
+      <div class="ml-10" style="color: white; margin-left: 60px; display: inline-block; vertical-align: middle;">Transactions</div> 
+    </a>
+    <div class="navbar-item" style="background-color: #3778c2; margin-left: 20px;">
+      <%= form_tag(destroy_user_session_path, method: :delete) do %>
+        <%= submit_tag 'Log Out', class: "button is-primary ml-10", style: "background-color: #3778c2;" %> 
+      <% end %>
+    </div>
+  </div>
+</nav>
+
+<div class = "img-amount-container">
+  <div class="trans-total-container">
+    <div class = "img-container">
+      <img class = "category-image" src= "<%= @category.icon %>" alt="<%= @category.name %>_image">
+    </div>
+    <p class="trans-name"><%= @category.name %></p>
+  </div>
+  <p class= "trans-amount">$<%= @category.total_amount %></p>
+</div>
+
+<ul id="transactions" class = "categories-ul">
+  <% @transactions.each do |transaction| %>
+    <li>
+      <p class = ""><%= transaction.name %></p>
+      <p class = "">$<%= transaction.amount %></p>
+    </li> 
+
+    <div class="button-container">
+    <%= button_to "Delete Transaction", category_exchange_path(@category, transaction), method: :delete, class: 'button is-light is-small delete_btn', style: 'box-shadow: 0 2px 4px #36363682; border-radius: 8px;margin-top:10px;' %> 
+ <%= link_to "Update Transaction", edit_category_exchange_path(@category, transaction), class: 'button is-light is-small delete_btn', style: 'box-shadow: 0 2px 4px #36363682; border-radius: 8px;' %>
+    </div> 
```

Comment:
- [ ] The transactions on the transactions index page should be redesigned to match the design. You should indicate the time the item was created and include some kind of image to maintain the positioning of elements as requested in the project.

  Your Implementation | Behance Design
  :---:|:---:
  ![image](https://github.com/AlyAtalla/budget-app/assets/54848363/bde8c418-545e-44f7-a678-b39ec60099ae) | ![image](https://github.com/AlyAtalla/budget-app/assets/54848363/ac0d7b31-5900-4564-a60c-5a456dca28f0)